app/src/main/java/com/example/peterchu/watplanner/home/HomeActivity.java

```diff
@@ -58,4 +61,27 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
         return super.onOptionsItemSelected(item);
     }
+
+    @Override
+    public void onRequestPermissionsResult(int requestCode,
+                                           String permissions[], int[] grantResults) {
+        switch (requestCode) {
```

Comment:
after pressing export, it will export the schedule if the user has the permissions. Otherwise, it will open up a permissions dialog, and on click of the permissions dialog, it will grand the permission and export the schedule